docs/pages/configuration/_partials/functions/create_deployments/example.mdx

```diff
@@ -0,0 +1,20 @@
+#### Example:
+
+```yaml
+images:
+  myImage:
+    dockerfile: ./Dockerfile
+
+pipelines:
+  create: |-
+     echo "Welcome to your custom create pipeline!"
+
+     # Exchange Dockerfile here (use --set-string if you want to force the field to be a string e.g. when setting labels to "true")
+     create_deployments myImage --set dockerfile=./Dockerfile.dev --set-string key1=value1
```

Comment:
deployments uses the deployment config, so we should use an example deployment here instead